sdk/core/azure-core/inc/azure/core/http/policies/policy.hpp

```diff
@@ -88,14 +90,14 @@ namespace Azure { namespace Core { namespace Http { namespace Policies {
      * @note See https://en.cppreference.com/w/cpp/chrono/duration.
      *
      */
-    std::chrono::milliseconds RetryDelay = std::chrono::seconds(4);
+    std::chrono::milliseconds RetryDelay = std::chrono::milliseconds(800);
 
     /**
      * @brief The maximum permissible delay between retry attempts.
      * @note See https://en.cppreference.com/w/cpp/chrono/duration.
      *
      */
-    std::chrono::milliseconds MaxRetryDelay = std::chrono::minutes(2);
+    std::chrono::milliseconds MaxRetryDelay = std::chrono::seconds(60);
```

Comment:
nit:
```suggestion
    std::chrono::milliseconds MaxRetryDelay = std::chrono::minutes(1);
```

Reply:
I specifically chose 60 seconds because that aligns with the text in the updated Azure standard for retry policies:
> YOU SHOULD implement the default policy with 3 retries with a 0.8s exponential (plus jitter) delay between tries with a maximum delay between tries of 60 seconds. 

While 1 minute is certainly 60 seconds, I think that the code should reflect the words of the standard.

Reply:
Is it documented somewhere? If so, can you add a link somewhere near the class definition, as a non-doxygen comment?

Reply:
Not yet, this is a discussed update to the guidelines amongst all the language architects. We should be able to update the docs as needed, once that has been ratified and published.

Reply:
FWIW, @tg-msft is on point for updating [this](https://azure.github.io/azure-sdk/general_azurecore.html#azurecore-http-retry-defaults) to include the 60 second max.